packages/forklift-console-plugin/src/modules/Overview/views/overview/tabs/Metrics/cards/ChartsCard.tsx

```diff
@@ -27,26 +27,45 @@ interface MigrationDataPoint {
   value: number;
 }
 
+enum TimeRangeOptions {
+  Last7Days,
+  Last31Days,
+  Last24H,
+}
+
```

Comment:
```suggestion
const TimeRangeOptions = {
  Last7Days: { label: 'Migrations (last 7 days)',  interval: { days: 7 }, bucket: { day: 1} },
  Last31Days:  { label: 'Migrations (last 31 days)',  interval: { days: 31 }, bucket: { day: 4} },
  Last24H:  { label: 'Migrations (last 24 hours)',  interval: { hours: 24 }, bucket: { hour: 4} },
}
```
can you try replace the ifs with a dictionry ?